src/rdo.rs

```diff
@@ -352,9 +352,15 @@ fn compute_tx_distortion<T: Pixel>(
   distortion
 }
 
-fn compute_rd_cost<T: Pixel>(fi: &FrameInvariants<T>, rate: u32, distortion: u64) -> f64 {
+fn bias_distortion(actual_distortion: f64, activity: u16, mean_activity: u16) -> f64 {
+  let s_1: f64 = 1_f64;
+  let s_0 = 0_f64;
+  actual_distortion * (s_1 * (mean_activity / activity) as f64 + s_0)
```

Comment:
Need a floor on `activity` to avoid divide-by-zero. Due to integer division, if `s_0 == 0` and `activity > mean_activity` then the distortion will be scaled by zero, which means rate is minimized at all costs.

Reply:
Yes, I observed that during the encodes and changed it locally. I'll push the changes soon.
```
actual_distortion * (s_1 * (mean_activity as f64 / activity as f64) + s_0)
```
This is what I'd done. I'll handle divide-by-zero error in the next revision.